components/post_view/post_list_ie/post_list_ie.jsx

```diff
@@ -522,6 +522,7 @@ export default class PostList extends React.PureComponent {
                     key={'post ' + (post.id || post.pending_post_id)}
                     post={post}
                     shouldHighlight={this.props.focusedPostId === post.id}
+                    previousPostId={posts[i + 1] ? posts[i + 1].id : ''}
```

Comment:
Will this actually be correct all the time? What if the post is skipped above? Do we need to account for that?

Reply:
@crspeller Thats a good question. I don't think this a problem as we have a different type of posts in place if we are skipping rendering but just in case i changed it to store non skipped posts here. Nice catch